verify_examples.sh

```diff
@@ -63,7 +63,7 @@ printf "Update done:\n\n"
 # Build directories that contain main package. These directories are different than
 # directories that contain go.mod files.
 printf "Build examples:\n"
-EXAMPLES=$(./get_main_pkgs.sh ./examples)
+EXAMPLES=$(find ./examples -type f -name go.mod -exec dirname {} \;)
```

Comment:
Similar here, this can be restored for this PR.

```suggestion
EXAMPLES=$(./get_main_pkgs.sh ./examples)
```